cmake/ConnectorName.cmake

```diff
@@ -11,7 +11,7 @@ IF(SIZEOF_VOIDP EQUAL 8)
   SET(IS64 1)
 ENDIF()
 
-SET (PLAFORM_NAME CMAKE_SYSTEM_NAME)
+SET (PLATFORM_NAME CMAKE_SYSTEM_NAME)
```

Comment:
Consider de-referencing ${CMAKE_SYSTEM_NAME}.

Reply:
Like this? https://github.com/mariadb-corporation/mariadb-connector-c/pull/170/commits/33da0ef5a24caa2348880f773d729d5999751b67

I have not tested this at all...